queries.sql

```diff
@@ -93,3 +93,73 @@ select owners.full_name as full_name,count(*) as count from animals
 join owners on animals.owner_id=owners.id
 group by owners.full_name
 order by count DESC;
+
+
+/* Vet clinic database: add "join table" for visits */
+
+/* Who was the last animal seen by William Tatcher? */
+SELECT animals.name as animal_name,
+vets.name as vet_name, 
+visits.visit_date 
+FROM animals
+JOIN visits on visits.animal=animals.id
+JOIN vets on vets.id=visits.vet
+where vets.name like 'William Tatcher'
+ORDER BY visits.visit_date DESC LIMIT 1;
+
+/* How many different animals did Stephanie Mendez see? */
+select COUNT(DISTINCT animals.name) from animals
+JOIN visits ON animals.id=visits.animal
+JOIN vets ON visits.vet=vets.id
+where vets.name like 'Stephanie Mendez';
+
+/* List all vets and their specialties, including vets with no specialties. */
+SELECT * from vets 
+LEFT JOIN specialties ON vets.id=specialties.vet
+LEFT JOIN species ON species.id=specialties.species
+
+/* List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020. */
+SELECT * FROM animals
+JOIN visits ON animals.id=visits.animal
+JOIN vets ON visits.vet=vets.id
+where vets.name like 'Stephanie Mendez'
+AND visits.visit_date between '2020/4/1' AND '2020/8/30';
+
+/* What animal has the most visits to vets? */
+select animals.name, count(visits.animal) from animals
+JOIN visits ON animals.id=visits.animal
+JOIN vets ON visits.vet=vets.id
+GROUP BY animals.name
+ORDER BY COUNT DESC LIMIT 1;
+
+/* Who was Maisy Smith's first visit? */
+select animals.name, vets.name as vet_name, visits.visit_date FROM animals
+JOIN visits ON animals.id=visits.animal
+JOIN vets ON visits.vet=vets.id
+WHERE vets.name like 'Maisy Smith' 
+ORDER BY visits.visit_date LIMIT 1;
+
+/* Details for most recent visit: animal information, vet information, and date of visit. */
+select animals.name as animal_name,
+vets.name as vet_name, 
+visits.visit_date 
+FROM animals
+JOIN visits ON animals.id=visits.animal
+JOIN vets ON visits.vet=vets.id
+ORDER BY visits.visit_date DESC LIMIT 1;
```

Comment:
- [ ] If you update `Plantmon's` visit to **_Maisy Smith_**, the accurate result for this query should be `Devimon` who visited Stephanie Mendez on May 4th, 2021. Kindly attach the correct result in your pull request description. 👍

Reply:
Indeed, 😀. Thank you.